src/Sentry/Internal/ApplicationVersionLocator.cs

```diff
@@ -9,15 +9,20 @@ internal static class ApplicationVersionLocator
 
         internal static string? GetCurrent(Assembly? asm)
         {
-            var version = asm?.GetNameAndVersion().Version;
+            if (asm is null)
+            {
+                return null;
+            }
+
+            var version = asm.GetNameAndVersion().Version;
 
             return !string.IsNullOrEmpty(version)
                    // If it really was on of the following, app would need to be set explicitly since these are defaults.
                    && version != "0.0.0"
                    && version != "1.0.0"
                    && version != "0.0.0.0"
                    && version != "1.0.0.0"
-                ? version
+                ? $"{asm.GetName().Name}@{version}"
```

Comment:
I think you don't need to check if the version is 0.0.0, 1.0.0 anymore.
@bruno-garcia  ping

Reply:
yeah, lets just alwayas send this.

need to update docs though

Reply:
Needs fixing here: https://github.com/getsentry/sentry-dotnet/blob/main/docs/manual/manual.md#release

Actually that could be deleted. It was before proper Sentry docs was added.

Needs fixing here: https://github.com/getsentry/sentry-docs/edit/master/src/platforms/common/configuration/releases.mdx